benchmarks/aggregate.py

```diff
@@ -36,6 +36,38 @@
 _markers = ('D', 'o', 's')
 
 
+class DatapointSelector:
+
+  def compile(row):
+    total_times = row['metrics']['total_time'] if 'total_time' in row[
+        'metrics'] else []
+    if len(total_times) <= 1:
+      return None
+    compile_and_run_time = total_times[0]
+    run_time = np.average(total_times[1:])
+    std = np.std(total_times[1:])
```

Comment:
Let's not compute std then -- AFAICT it's unused.

Reply:
Done.